wgpu/backends/wgpu_native/_api.py

```diff
@@ -499,7 +508,13 @@ def configure(
             str_val = enum_int2str["CompositeAlphaMode"][int_val]
             capable_alpha_modes.append(str_val.lower())
 
-        # H: void f(WGPUSurfaceCapabilities capabilities)
+        # The format is used as-is
+        if format is None and capabilities.formatCount > 0:
+            format = enum_int2str["TextureFormat"][capabilities.formats[0]]
+        else:
+            raise RuntimeError("No formats supported by the surface.") # doesn't really make sense, maybe should be higher up.
```

Comment:
I am getting capabilities with Null pointers for formats and preset modes when I use Vulkan backend(default). 

tried today with D3D12 and OpenGL and it populates those fields. Don't think this anything in my control right now, so I will use the other backends with `WGPU_BACKEND_TYPE`